core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/extensions/replication/pg-new/repmgr-register-standby.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: PostgreSQL | Extensions | repmgr | Clone standby node with repmgr
+  become_user: postgres
+  command: >-
+    {{ new_version.repmgr.bin_dir[ansible_os_family] }}/repmgr standby clone
+      -h {{ pg_primary_node }}
+      -U {{ specification.extensions.replication.privileged_user_name }}
+      -d {{ specification.extensions.replication.repmgr_database }} -p 5432 --force
+  register: repmgr_standby_clone
+  changed_when: repmgr_standby_clone.rc == 0
+
+- name: PostgreSQL | Start PostgreSQL {{ new_version.pg.version }} service
+  systemd:
+    name: "{{ pg.service_name[ansible_os_family] }}"
```

Comment:
How about waiting for the instantiated service on Ubuntu to be started?

Something like:
```
- name: Ensure PostgreSQL instantiated service is running
  systemd:
    name: "{{ pg.instantiated_service_name[ansible_os_family] }}"
    state: started
  when: pg.instantiated_service_name[ansible_os_family] != None
```